src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -478,6 +478,14 @@ private function getProperty(PropertyMetadata $propertyMetadata, string $propert
         if (null !== $type && !$type->isCollection() && (null !== $className = $type->getClassName()) && $this->resourceClassResolver->isResourceClass($className)) {
             $propertyData['owl:maxCardinality'] = 1;
         }
+        // Add doc informations
+        $docInformations = $propertyMetadata->getAttributes()['doc_informations'] ?? [];
+        foreach ($docInformations as $infoKey => $infoValue) {
+            if ('@id' === $infoKey || '@type' === $infoKey) {
```

Comment:
Why excluding those keys (maybe can we just let the user override this if he wants to)?